_projects/2022/100385774/100385774.Rmd

```diff
@@ -0,0 +1,902 @@
+  annotate("segment", x=2021.45, xend= 2021.6, y=31, yend=40, color="grey") +
+  annotate("segment", x=2021.6, xend= 2021.9, y=40, yend=40, color="grey") +
+  #LinkedIn
+  annotate("segment", x=2021.15, xend= 2021.45, y=28, yend=28, color="grey") +
+  annotate("segment", x=2021.45, xend= 2021.6, y=28, yend=34, color="grey") +
+  annotate("segment", x=2021.6, xend= 2021.9, y=34, yend=34, color="grey") +
+  #Snapchat
+  annotate("segment", x=2021.15, xend= 2021.45, y=25, yend=25, color="grey") +
+  annotate("segment", x=2021.45, xend= 2021.6, y=25, yend=29, color="grey") +
+  annotate("segment", x=2021.6, xend= 2021.9, y=29, yend=29, color="grey") +
+  #Twitter
+  annotate("segment", x=2021.15, xend= 2021.9, y=24, yend=24, color="grey") +
+  #Whatsapp
+  annotate("segment", x=2021.15, xend= 2021.45, y=23, yend=23, color="grey") +
+  annotate("segment", x=2021.45, xend= 2021.6, y=23, yend=18, color="grey") +
+  annotate("segment", x=2021.6, xend= 2021.9, y=18, yend=18, color="grey") +
+  #TikTok
+  annotate("segment", x=2021.15, xend= 2021.45, y=21, yend=21, color="grey") +
+  annotate("segment", x=2021.45, xend= 2021.6, y=21, yend=15, color="grey") +
+  annotate("segment", x=2021.6, xend= 2021.9, y=15, yend=15, color="grey") +
+  #Reddit
+  annotate("segment", x=2021.15, xend= 2021.45, y=18, yend=18, color="grey") +
+  annotate("segment", x=2021.45, xend= 2021.6, y=18, yend=10, color="grey") +
+  annotate("segment", x=2021.6, xend= 2021.9, y=10, yend=10, color="grey") +
+  #Nextdoor
+  annotate("segment", x=2021.15, xend= 2021.45, y=13, yend=13, color="grey") +
+  annotate("segment", x=2021.45, xend= 2021.6, y=13, yend=6, color="grey") +
+  annotate("segment", x=2021.6, xend= 2021.9, y=6, yend=6, color="grey") +
+  
+  annotate("text", label= "PEW RESEARCH CENTER", x=2012, y=0, vjust=16.5, 
+           hjust=0.15, family = "Franklin Gothic", size=2.8) +
+  labs(title = paste(" Growing share of Americans say that they use YouTube;\n", 
+"Facebook remains one of the most widely used online\n",
+"platforms among U.S. adults"),
+       subtitle = "% of U.S. adults who say they ever use ...",
+       caption = paste("Note: Respondents who did not give an answer are not shown. Pre-2018 telephone poll",
+       "data is not available for Youtube, Snapchat and Whatsapp; pre-2019 telephone poll data is",
+       "not available for Reddit. Pre-2021 telephone poll data is not available for TikTok. Trend",
+       "data is not available for Nextdoor.",
+       "Source: Survey of U.S. adults conducted Jan. 25-Feb. 8, 2021.", 
+       '"Social media use in 2021"',  sep="\n")) +
+coord_fixed(xlim=c(2012,2021), clip = 'off')
+
+```
+
+### Addendum 2: Source and available data
+
+Article: https://www.pewresearch.org/internet/2021/04/07/social-media-use-in-2021/
+
+Survey data: https://www.pewresearch.org/internet/wp-content/uploads/sites/9/2021/04/PI_2021.04.07_Social-Media-Use_TOPLINE.pdf
```

Comment:
Please remove this and integrate these two links in the first paragraphs of the article with Markdown links (i.e. [some text](url)).